### DISCUSSION TITLE
Getting exception as Invalid arguments were passed PythonDecoratedOperator

### DISCUSSION THREAD

**Discussion by leoliveroso** [apache/airflow#30592] [Q&A]
I am getting exception from baseoperator. Py as Invalid arguments were passed to PythonDecoratedOperator while using partial() and expand() method.  Any idea to fix this?

<img width="1145" alt="image" src="https://user-images.githubusercontent.com/39195354/231311299-1dcd7ab0-e9e4-4355-a738-3ae00a7f32b9.png">

Syntax used for constraints as below.

<img width="539" alt="image" src="https://user-images.githubusercontent.com/39195354/231311217-6ae4beaf-fd10-4180-9a13-0d2d4f3d5cf5.png">

`list_partitions = [str(i) for i in range(PARAMS_DEFINITION['num_workers'])]
    render_template_task = render_template.partial(path_dest=PARAMS_DEFINITION['folder_dir']['pdf']).expand(
        counter=list_partitions
    )
`

I am using Airflow 2.3.2

**Comment by potiuk** [comment]
Looks like your DAG has default_args whch has "concurency" defined - this gets applied to all tasks, and your task does not accept concurrency parameter.
